### ISSUE TITLE
Add actual 'normalize' method

### ISSUE THREAD

**Issue by floscha** [floscha/tabular-dataset#24]
Add a transformation to `NumericalColumns` that works like scitkit-learn's [normalize](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.normalize.html) and rename the existing `normalize` method to `scale`.

**Comment by floscha** [comment]
Resolved with #53.
